cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/LocationCategoryEndpointInput.java

```diff
@@ -48,7 +48,7 @@ public LocationCategoryEndpointInput officeId(String officeId) {
     @Override
     protected HttpRequestBuilder addInputParameters(HttpRequestBuilder httpRequestBuilder) {
         return httpRequestBuilder.addQueryParameter(CATEGORY_ID_QUERY_PARAMETER, categoryId)
-            .addQueryParameter(OFFICE_QUERY_PARAMETER, officeId)
-            .addQueryHeader(ACCEPT_QUERY_HEADER, "application/json");
+                                 .addQueryParameter(OFFICE_QUERY_PARAMETER, officeId)
+                                 .addQueryHeader(ACCEPT_QUERY_HEADER, "application/json");
```

Comment:
I think Bryson introduced constants for this

Reply:
He only had it in a single class. I refactored into a constants class.